src/vivarium/examples/boids/visualization.py

```diff
@@ -1,42 +1,43 @@
-# mypy: ignore-errors
 import matplotlib.pyplot as plt
 from matplotlib.animation import FuncAnimation
 
+from vivarium import InteractiveContext
 
-def plot_boids(simulation, plot_velocity=False):
+
+def plot_boids(simulation: InteractiveContext, plot_velocity: bool=False) -> None:
     width = simulation.configuration.field.width
     height = simulation.configuration.field.height
     pop = simulation.get_population()
 
-    plt.figure(figsize=[12, 12])
+    plt.figure(figsize=(12, 12))
     plt.scatter(pop.x, pop.y, color=pop.color)
     if plot_velocity:
         plt.quiver(pop.x, pop.y, pop.vx, pop.vy, color=pop.color, width=0.002)
     plt.xlabel("x")
     plt.ylabel("y")
-    plt.axis([0, width, 0, height])
+    plt.axis((0, width, 0, height))
     plt.show()
 
 
-def plot_boids_animated(simulation):
+def plot_boids_animated(simulation: InteractiveContext) -> FuncAnimation:
```

Comment:
Is this an `InteractiveContext` or a `SimulationContext`?

Reply:
I'm not really sure tbh. The example docs is all about interactive, but maybe it could be used in a simulation context?